Modules/xSharePoint/DSCResources/MSFT_xSPVisioServiceApp/MSFT_xSPVisioServiceApp.schema.mof

```diff
@@ -0,0 +1,18 @@
+/*
+**Description**
+
+This resource is responsible for creating Visio Graphics Service Application instances within the local SharePoint farm.
+The resource will provision and configure the Visio Graphics Service Application.
+**Example**
+    xSPVisioServiceApp VisioServices
+    {
+        Name = "Visio Graphics Service Application"
+        ApplicationPool = "SharePoint Web Services"
+    }
+*/
+[ClassVersion("1.0.0.0"), FriendlyName("xSPVisioServiceApp")]
+class MSFT_xSPWebApplication : OMI_BaseResource
```

Comment:
Looks fine in the BCS service app - you should get a complaint when importing the DSC resource to generate a MOF file if it finds an object that doesn't match the folder name, so it should be pretty obvious where this kind of thing falls out when we run it.